test/com/puppetlabs/puppetdb/test/scf/storage.clj

```diff
@@ -510,6 +627,8 @@
     (is (= (query-to-vec ["SELECT COUNT(*) as c FROM catalog_resources"])
            [{:c 3}]))))
 
+;; FIXME: should this and delete-certname-facts! go away once the new
```

Comment:
Yes, that's what I had in mind when I wrote the comment at least (before I wrote the tests).